### PR TITLE
kubectl v1.1.2

### PR DESCRIPTION
**Pull request by antoineco**

### CODE REVIEW

File: Library/Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -1,11 +1,11 @@
 class KubernetesCli < Formula
-  desc "Command-line tool for kubernetes, a cluster manager for Docker"
+  desc "Command-line tool for Kubernetes, an open source system for managing containerized applications across multiple hosts"
```
Comment (bfontaine) [inline Library/Formula/kubernetes-cli.rb]: Please shorten this; `kubernetes-cli: <desc>` should fit under 80 characters.

**Comment by antoineco** [comment]
@bfontaine thanks for the info :+1:  I shortened it

**Comment by bfontaine** [comment]
@antoineco The new one was too long but the existing one is OK; you can leave it as is :wink:

**Comment by antoineco** [comment]
I have one concern regarding the previous description because it's not accurate. Kubernetes is not a cluster manager for "Docker", it's a cluster manager for containerized applications in general.

**Comment by bfontaine** [comment]
Makes sense; thank you for your contribution! :tada:

**Pull request closed without merging**